client/src/components/QA/QAItem.jsx

```diff
@@ -17,8 +23,85 @@ export default function QAItem({ question, allAnswers }) {
       totalAsToRender.push(arrayOfAnswers[i]);
     }
   }
-  // console.log(totalAsToRender);
-  // console.log(moreAsClicked && totalAsToRender[0] !== undefined);
+
+  const handleQuestionHelpful = (category, ID) => {
+    axios({
+      method: 'PUT',
+      url: `http://localhost:3000/qa/questions/${ID}/${category}`,
+    })
+      .then((response) => {
+        console.log(response.status);
+        const copyAllQuestions = [...globalQData];
+        for (let i = 0; i < copyAllQuestions.length; i += 1) {
+          if (copyAllQuestions[i].question_id === ID) {
+            copyAllQuestions[i].question_helpfulness += 1;
+            setGlobalQData(copyAllQuestions);
+          }
+        }
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  const handleHelpfulAnswer = (category, ID) => {
+    axios({
+      method: 'PUT',
+      url: `http://localhost:3000/qa/answers/${ID}/${category}`,
+    })
+      .then((response) => {
+        console.log(response.status);
+        const copyAllAnswers = [...allAnswers];
+        for (let i = 0; i < copyAllAnswers.length; i += 1) {
+          if (copyAllAnswers[i].question === question.question_id.toString()) {
+            for (let j = 0; j < copyAllAnswers[i].results.length; j += 1) {
+              if (copyAllAnswers[i].results[j].answer_id === ID) {
+                copyAllAnswers[i].results[j].helpfulness += 1;
+                setGlobalAData(copyAllAnswers);
```

Comment:
As a suggestion, it might be good to change the variable 'setGlobalAData' to what I assume is 'setGlobalAnswerData'. It may help readability and prevent you from accidentally typing 'a' instead of 'A'. But if I'm wrong my apologies!